### PR TITLE
feat: Add support for Python 3.11

### PR DESCRIPTION
**Pull request by rmortes**
Currently only Django 4.1 supports Python 3.11. I've added this case to the CI and it works.

I don't know it there's more to this PR

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -74,9 +74,10 @@ envlist =
     mypy
     # tests against released versions
     py{36,37,38,39,310}-dj{22,31,32,40,41}-redislatest
+    py{311}-dj{41}-redislatest
     # tests against unreleased versions
-    py310-dj40-redismaster
-    py310-djmain-redis{latest,master}
+    py311-dj40-redismaster
```
Comment (WisdomPill) [inline setup.cfg:79]: I think you wanted to test against `django 4.1`

```suggestion
    py311-dj41-redismaster
```

File: setup.cfg [hunk 2]
```diff
@@ -74,9 +74,10 @@ envlist =
     mypy
     # tests against released versions
     py{36,37,38,39,310}-dj{22,31,32,40,41}-redislatest
+    py{311}-dj{41}-redislatest
```
Comment (WisdomPill) [inline setup.cfg:77]: why is this here?

**Review by WisdomPill — APPROVED** [review]
Thanks for your contribution and quick PR!
Help is always welcome here!

I left a couple of comments

**Comment by WisdomPill** [comment]
also please make towwncrier happy by adding a changelog file

**Comment by rmortes** [comment]
Hi @WisdomPill!

Sorry I didn't get to you sooner, I *really* should check out my github notifications more often. I've seen you closed my issue and added 3.11 support in a new PR. Thank you very much for your great work and for your review, even if I didn't fix my code - and sorry for that, I'm still new to open source collaborations

**Comment by WisdomPill** [comment]
no worries, you're always welcome to open other PRs and contribute in the future! 😁
